website/docs/topics/non-openai-models/cloud-gemini_vertexai.ipynb

```diff
@@ -0,0 +1,808 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"https://colab.research.google.com/github/microsoft/autogen/blob/main/notebook/oai_chatgpt_gpt4.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+   ]
```

Comment:
Remove this cell. The links will be automatically added when building the website.

Reply:
Done 🙂